packages/driver/src/cypress/error_messages.coffee

```diff
@@ -182,6 +182,17 @@ module.exports = {
       Fix this problem, or use {force: true} to disable error checking.
 
       https://on.cypress.io/element-cannot-be-interacted-with
+      """
+    pointer_events_none: (obj) ->
+      """
+        #{cmd(obj.cmd)} failed because this element:
+
+        #{obj.element}
+
+        has style 'pointer-events:none'#{if obj.elementInherited then ", inherited from this element:" else ""}
+
+        #{if obj.elementInherited then obj.elementInherited else ""}
```

Comment:
You're missing the `https://on.cypress.io/element-cannot-be-interacted-with` at the bottom of this error - and additionally we'll need to update the docs to account for this situation.

You're also missing the verbiage about bypassing this error with `{ force: true }`.

Lastly, i would add a small bit explaining to the user that `pointer-events: none` prevents this element from receiving any events or being able to be interacted with the mouse, keyboard, etc which is **why** we cannot interact with it.